src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+
+type Props = {
+  filterBy: string,
+  setFilterBy: (value: string) => void
+};
+
+const navigation = ['All', 'Active', 'Completed'];
+
+export const Footer: React.FC<Props> = ({ filterBy, setFilterBy }) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        3 items left
```

Comment:
according to the task requirements, you should implement a filter, including this counter
it should show real count of active (uncompleted) todos